crates/cli/src/commands/doctor.rs

```diff
@@ -278,62 +275,37 @@ Check that the homeserver is running."#
                 Err(e) => error!(
                     r#"❌ Can't reach the homeserver at "{whoami}".
 
-Error details: {e}
-"#
-                ),
-            }
-
-            // Try to reach the admin API on an unauthorized endpoint
-            let server_version = hs_api.join("/_synapse/admin/v1/server_version")?;
-            let result = http_client.get(server_version.as_str()).send_traced().await;
-            match result {
-                Ok(response) => {
-                    let status = response.status();
-                    if status.is_success() {
-                        info!(r#"✅ The Synapse admin API is reachable at "{server_version}"."#);
-                    } else {
-                        error!(
-                            r#"❌ A Synapse admin API endpoint at "{server_version}" replied with {status}.
-Make sure MAS can reach the admin API, and that the homeserver is running.
-"#
-                        );
-                    }
-                }
-                Err(e) => error!(
-                    r#"❌ Can't reach the Synapse admin API at "{server_version}".
-Make sure MAS can reach the admin API, and that the homeserver is running.
-
```

Comment:
Why is this no longer useful? Doesn't MAS still need to use Synapse's Admin API in several cases?

Reply:
It's using a dedicated API now, and doesn't have full admin access anymore

Reply:
This is also now just testing for the dedicated MAS API, which should cover both the 'it's unreachable' and the 'it's the wrong secret' cases